patches/components-embedder_support-user_agent_utils.cc.patch

```diff
@@ -15,3 +15,19 @@ index 64f9e50f60186a06d1b0998f1f2d5b83ca864cdf..c1f09e4c3cf7f8c67b4b6252e67b0214
            : major_version;
  
    return GenerateBrandVersionList(major_version_number, brand, brand_version,
+@@ -653,6 +654,7 @@ blink::UserAgentMetadata GetUserAgentMetadata(const PrefService* pref_service,
+   metadata.full_version = std::string(version_info::GetVersionNumber());
+   metadata.architecture = GetCpuArchitecture();
+   metadata.model = BuildModelInfo();
++  BRAVE_GET_USER_AGENT_METADATA
```

Comment:
`#define HardwareModelName() HardwareModelName() == "" ? "" : ""`

Reply:
or something like `#define GetAndroidBuildCodename() GetAndroidBuildCodename() == "" && "" ==`

Reply:
Fixed in 6d3da125ac49fcb80322fb3a175d99c6288ea83f.